runxtb.sh

```diff
@@ -630,11 +637,19 @@ write_submit_script ()
 			# Loading the modules should take care of everything except threads
 			# Export current (at the time of execution) MODULEPATH (to be safe, could be set in bashrc)
 			export MODULEPATH="$MODULEPATH"
-			module load ${load_modules[*]} 2>&1 || exit 1
+			EOF
+      if [[ "$queue" =~ [Rr][Ww][Tt][Hh] ]] ; then
+        echo "module unload intel" >&9
+      fi
+      for mod in "${load_modules[@]}" ; do
+        cat >&9 <<-EOF
+				module load "${mod}"
+				EOF
```

Comment:
this can probably just be `module load "${mod}$ >&9`

Reply:
Probably this would work as well to replace 644-648:
```bash
for mod in "${load_modules[@]}" ; do
  echo "module load ${mod}"
done
```